sormas-app/app/src/main/java/de/symeda/sormas/app/backend/sample/Sample.java

```diff
@@ -99,6 +100,9 @@
     @Column(length = 512)
     private String labDetails;
 
+    @Enumerated(EnumType.STRING)
+    private SamplePurpose samplePurpose;
```

Comment:
Also needs to be added to the database via DatabaseHelper. See https://github.com/hzi-braunschweig/SORMAS-Project/blob/development/GUIDE_ADD_NEW_FIELD.md#iv-adding-the-field-to-the-sormas-android-app

Reply:
i am getting this error after "Database upgrade failed for version 167: Could not run raw execute statement ALTER TABLE samples ADD COLUMN samplePurpose varchar(255);"

Reply:
@MartinWahnschaffeSymeda please comment on this

Reply:
@opatajoshua Sorry, I have asked for clarification via skype: I will need to see your source code you hava added to the DatabaseHelper.onUpgrade method. The branch does not seem to have it yet: https://github.com/GCNETGHANA/SORMAS-Project/blob/feature-1109-bed-side-lab-testing/sormas-app/app/src/main/java/de/symeda/sormas/app/backend/common/DatabaseHelper.java

Reply:
ok let me push what i have now

Reply:
Have you pushed it? I don't see any new commits related to this

Reply:
Has been pushed now

Reply:
Ah, I see. The case has to be 166, not 167. 
Explanation: The DATABASE_VERSION ist the new version (167 now - this part is correct).
The number in the switch case is the previous version, so 166. It tells the system what to do when moving from the previous version to the new one.
When both numbers are the same, this will always be called. So your statement will have worked the first time, but no more, once you have started the app the second time.

Reply:
changing that

Reply:
changed but still had the error. the message is not detailed. i had to uninstall before rerunning the app